pkg/cmd/server/bootstrappolicy/policy_test.go

```diff
@@ -106,6 +114,69 @@ func TestBootstrapClusterRoles(t *testing.T) {
 	testObjects(t, list, "bootstrap_cluster_roles.yaml")
 }
 
+func findGroupForPolicyRules(groupName string, rules []rbacv1.PolicyRule) *rbacv1.PolicyRule {
```

Comment:
Use the rbac Covers function to confirm that the cluster reader cluster role covers the two rules.